### PR TITLE
Initial v1 docs

### LINKED ISSUE #11: [KDOCS-9] Getting Started guide
Walk a user through a basic setup with a reward of each type, basic multipliers, etc

<sub>From [SyncLinear.com](https://synclinear.com) | [KDOCS-9](https://linear.app/buape/issue/KDOCS-9/getting-started-guide)</sub>

### PR DESCRIPTION
**Pull request by thewilloftheshadow**
This PR adds the basic set of commands and feature structure

https://kiai-docs-git-v1-thewilloftheshadow.vercel.app/

Implements the structure for #11 

Closes KDOCS-3
Closes KDOCS-5
Closes KDOCS-7
Closes KDOCS-10
Closes KDOCS-11
Closes KDOCS-12
Closes KDOCS-15
Closes KDOCS-16
Closes KDOCS-22

### CODE REVIEW

**Review by daksh-gg — COMMENTED** [review]
very nice 👍

File: pages/_meta.json
Comment (net-tech) [inline pages/_meta.json:1]: Can you add a link to the main website?

File: pages/commands/debug.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Ping Command
```
Comment (net-tech) [inline pages/commands/debug.mdx:1]:
```suggestion
# Debug Command
```

File: pages/commands/debug.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+This command is available to users with the Manage Server permission.
+
+## Description
+
+The `/debug` command generates a report with debug information for our team to use in troubleshooting. This report includes the following information:
+
+-   Guild Info
+-   User Info
+-   Your Leveling Info
+-   The Bot's Shard Information
```
Comment (net-tech) [inline pages/commands/debug.mdx:20]:
```suggestion
-   Server Information such as the server ID, the bot's permissions, and the server's settings
-   Information about you, such as your Discord ID, permissions in the server, and level
-   General bot information such as the ping and version
```

File: pages/commands/leaderboard.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+## Syntax
+
+```
+/leaderboard
+```
+
+## Permissions
+
+This command is available to anyone.
```
Comment (net-tech) [inline pages/commands/leaderboard.mdx:11]:
```suggestion
This command is available to everyone.
```

File: pages/commands/leaderboard.mdx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+## Description
+
+The `/leaderboard` command will send the top 10 members with the most XP in the server, and provide a link to the full leaderboard located at https://kiai.wtf
```
Comment (net-tech) [inline pages/commands/leaderboard.mdx:15]:
```suggestion
The `/leaderboard` command will show the top 10 members with the most XP in the server, and provide a link to the full leaderboard located at https://kiai.wtf.
```

File: pages/commands/level-up.mdx [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+Running this command will open a popup modal that has fields for each element of the level up message.
+
+### Placeholders
+
+These are the available placeholders for you to use in the level up messages:
+
+| Placeholder            | Description              | Example                                                                                    |
+| ---------------------- | ------------------------ | ------------------------------------------------------------------------------------------ |
+| `{user}`               | The user who leveled up  | `@User#0001`                                                                               |
+| `{user.mention}`       | The user who leveled up  | `@User#0001`                                                                               |
```
Comment (net-tech) [inline pages/commands/level-up.mdx:26]:
```suggestion
| `{user.mention}`       | A @mention of the user who leveled up  | `@User#0001`                                                                               |
```

File: pages/commands/level-up.mdx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+| `{user.id}`            | The user's ID            | `123456789012345678`                                                                       |
```
Comment (net-tech) [inline pages/commands/level-up.mdx:27]:
```suggestion
| `{user.id}`            | The user's Discord ID            | `123456789012345678`                                                                       |
```

File: pages/commands/level-up.mdx [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+| `{user.tag}`           | The user's tag           | `User#0001`                                                                                |
+| `{user.username}`      | The user's username      | `User`                                                                                     |
+| `{user.discriminator}` | The user's discriminator | `0001`                                                                                     |
+| `{guild.id}`           | The guild's ID           | `123456789012345678`                                                                       |
+| `{guild.name}`         | The guild's name         | `My Server`                                                                                |
+| `{guild.icon}`         | The guild's icon         | `https://cdn.discordapp.com/icons/123456789012345678/abcdef1234567890abcdef1234567890.png` |
```
Comment (net-tech) [inline pages/commands/level-up.mdx:33]:
```suggestion
| `{guild.id}`           | The server's ID           | `123456789012345678`                                                                       |
| `{guild.name}`         | The server's name         | `My Server`                                                                                |
| `{guild.icon}`         | The server's icon         | `https://cdn.discordapp.com/icons/123456789012345678/abcdef1234567890abcdef1234567890.png` |
```

File: pages/commands/ping.mdx [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+/ping
+```
+
+## Permissions
+
+This command is available to everyone.
+
+## Description
+
+The `/ping` command displays the current connection speed of the bot to Discord.
```
Comment (net-tech) [inline pages/commands/ping.mdx:15]:
```suggestion
The `/ping` command displays the latency between the bot and Discord.
```

File: pages/commands/rank.mdx [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+This command is available to everyone.
+
+## Description
+
+The `/rank` command displays your current rank, the amount of experience you have, and the number of messages you have sent.
+
+<Callout emoji="⚠️">
+	We're still working on the design for our rank card, the current message is
+	just a placeholder.
+</Callout>
```
Comment (net-tech) [inline pages/commands/rank.mdx:22]:
```suggestion
<Callout type="info" emoji="ℹ️">
	We're still working on the design for our rank card, the current message is
	just a placeholder.
</Callout>
```

File: pages/features/kiai.wtf.mdx [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+
+Kiai.wtf is our leaderboard site that provides leaderboards for all guilds that Kiai is in.
+
+## URL Format
+
+Most leaderboards hosted at the following address:
+```
+https://kiai.wtf/guild/608098293498511360
+```
+where 608098293498511360 is the ID of the guild you are viewing the leaderboard for.
```
Comment (net-tech) [inline pages/features/kiai.wtf.mdx:11]:
```suggestion
where 608098293498511360 is the ID of the server you are viewing the leaderboard for.
```

**Review by net-tech — CHANGES_REQUESTED** [review]

File: pages/_meta.json [hunk 11]
```diff
@@ -1,14 +1,11 @@
...
-    "type": "page",
-    "href": "https://kiaibot.com"
-  },
-  "about": {
-    "title": "About",
-    "type": "page"
-  }
+	"main-site": {
+		"type": "page",
+		"title": "Main Site",
```
Comment (net-tech) [inline pages/_meta.json:4]:
```suggestion
		"title": "Website",
```

**Review by net-tech — APPROVED** [review]
Make the change I commented, making as approved because im going to sleep

**Comment by thewilloftheshadow** [comment]
Two approvals outside of Github's system

**Pull request merged**